docs/migrating-to-v1.rst

```diff
@@ -0,0 +1,162 @@
+Migrating to Rasterio 1.0
+=========================
+
+
+I/O Operations
+--------------
+
+Methods related to reading band data and dataset masks have changed in 1.0.
+
+
+Deprecated: ``rasterio.drivers()``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Previously users could register GDAL's drivers and open a datasource with:
+
+.. code-block:: python
+
+    import rasterio
+
+    with rasterio.drivers():
+
+        with rasterio.open('tests/data/RGB.byte.tif') as src:
+            pass
+
+but Rasterio 1.0 contains more interactions with GDAL's environment, so
+``rasterio.drivers()`` has been replaced with:
+
+.. code-block:: python
+
+    import rasterio
+    import rasterio.env
+
+    with rasterio.env.Env():
```

Comment:
Is this different than `with rasterio.Env()`? Or are they equivalent. In #760 everything was set to just `rasterio.Env()` and wanted to make sure that is right.